core/settings.py

```diff
@@ -266,6 +266,7 @@ def context_processors():
 )
 default_settings = dj_database_url.parse(DATABASE_URL)
 default_settings["ENGINE"] = "django.contrib.gis.db.backends.postgis"
+default_settings["OPTIONS"] = {"options": "-c search_path=public,warehouse"}
```

Comment:
```suggestion
default_settings = {
  **dj_database_url.parse(DATABASE_URL),
  "ENGINE": "django.contrib.gis.db.backends.postgis",
  "OPTIONS": {"options": "-c search_path=public,warehouse"}
}
```

ça peut être écrit un peu plus lisiblement comme ça je pense 
```